### PR TITLE
build: script to update internal dependency versions

### PR DESCRIPTION
**Pull request by dobladez**
Release-As: 0.1.0

### What does this PR do?
Force version number for first release of soql-common.
Also: script to update version of internal dependencies in the monorepo

### CODE REVIEW

File: .github/workflows/release-soql-common.yml [hunk 1]
```diff
@@ -30,12 +30,12 @@ jobs:
...
         run: |
-         SOQL_MODEL_VERSION=${{ steps.release.outputs.major}}.${{ steps.release.outputs.minor }}.${{ steps.release.outputs.patch}}
-         sed -i.back -E 's/"@salesforce\/soql-common": +"[^"]+"/"@salesforce\/soql-common": "'$SOQL_MODEL_VERSION'"/g' packages/*/package.json
+         SOQL_COMMON_VERSION=${{ steps.release.outputs.major}}.${{ steps.release.outputs.minor }}.${{ steps.release.outputs.patch}}
+         sed -i.back -E 's/"@salesforce\/soql-common": +"[^"]+"/"@salesforce\/soql-common": "'$SOQL_COMMON_VERSION'"/g' packages/*/package.json
          if ! git diff --exit-code; then
            git config --local user.email "$(git log --format='%ae' HEAD^!)"
            git config --local user.name "$(git log --format='%an' HEAD^!)"
-           git commit -a -m "chore: bump soql-common version on soql-builder-ui to $SOQL_MODEL_VERSION"
+           git commit -a -m "chore: bump soql-common version dependents to $SOQL_MODEL_VERSION"
```
Comment (jgellin-sf) [inline .github/workflows/release-soql-common.yml:38]: Should this be `$SOQL_COMMON_VERSION`?

File: scripts/update-interdependency-versions.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ) {
+      package.dependencies[packageName] = package2version[packageName];
+      fs.writeFile(
+        packageFile,
+        JSON.stringify(package, null, 2),
+        'utf8',
+        (err) => {
+          if (err) throw err;
+          console.log(
+            `updated ${packageName} dependency version on ${pacakgeFile}`
```
Comment (jgellin-sf) [inline scripts/update-interdependency-versions.js:34]: TYPO
`pacakgeFile` -> `packageFile`

**Review by jgellin-sf — CHANGES_REQUESTED** [review]
Couple of typos

**Comment by dobladez** [comment]
Superseded by https://github.com/forcedotcom/soql-tooling/pull/208

**Pull request closed without merging**